dtest.py

```
@@ -100,6 +102,14 @@
 # set python-driver log level to INFO by default for dtest
 logging.getLogger('cassandra').setLevel(logging.INFO)
 
+
+def get_sha(repo_dir):
+    try:
+        return "git:{}".format(subprocess.check_output(['git', 'rev-parse', 'HEAD'], cwd=repo_dir).strip())
+    except CalledProcessError:
```

I approve of this for sure: https://github.com/riptano/cassandra-dtest/pull/1095/commits/b301da74cccedb7dd3cb89d8c86551dc6501d2c6#diff-4cbdcd9896ce5009acd37ed830ce8959R114

I'm not sure about catching `OSError`, and I've left a comment to that effect.

> small downside here is if the git output changes this could break in the future, but not a huge risk imo.

Yeah, I agree. Having to deal with different `git` versions should happen pretty infrequently, and it'll fail loudly, so I'm good wiht it.